### PR TITLE
Reduce scope of variables

### PR DESCRIPTION
**Pull request by thatsIch**

### CODE REVIEW

**Comment by yueh** [comment]
Out of curiosity did you do it manually or IDE support for it?

**Comment by thatsIch** [comment]
95% IDE, 5% me for prettifying

File: src/main/java/appeng/util/Platform.java [hunk 1]
```diff
@@ -187,9 +187,6 @@ public static String formatPowerLong( long n, boolean isRate )
 		PowerUnits displayUnits = AEConfig.instance.selectedPowerUnit();
```
Comment (orod-org) [inline src/main/java/appeng/util/Platform.java]: MINOR Unwritten public or protected field: appeng.core.AEConfig.instance [rule](http://nemo.sonarqube.org/coding_rules#rule_key=findbugs%3AUWF_UNWRITTEN_PUBLIC_OR_PROTECTED_FIELD)

**Comment by orod-org** [comment]
SonarQube analysis reported 5 issues:
* MINOR 3 minor
* INFO 2 info

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MINOR [Unwritten public or protected field: appeng.container.AEBaseContainer.powerSrc](https://github.com/AppliedEnergistics/Applied-Energistics-2/blob/0387d30d1cbb0c86a3bc5997938fa27e6f831cea/src/main/java/appeng/container/AEBaseContainer.java#L855) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=findbugs%3AUWF_UNWRITTEN_PUBLIC_OR_PROTECTED_FIELD)
* MINOR [Unwritten public or protected field: appeng.container.AEBaseContainer.cellInv](https://github.com/AppliedEnergistics/Applied-Energistics-2/blob/0387d30d1cbb0c86a3bc5997938fa27e6f831cea/src/main/java/appeng/container/AEBaseContainer.java#L855) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=findbugs%3AUWF_UNWRITTEN_PUBLIC_OR_PROTECTED_FIELD)
* INFO [Unread public/protected field: appeng.core.AEConfig.minMeteoriteDistanceSq](https://github.com/AppliedEnergistics/Applied-Energistics-2/blob/0387d30d1cbb0c86a3bc5997938fa27e6f831cea/src/main/java/appeng/core/AEConfig.java#L75) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=findbugs%3AURF_UNREAD_PUBLIC_OR_PROTECTED_FIELD)
* INFO [Unread public/protected field: appeng.me.cache.PathGridCache.channelPowerUsage](https://github.com/AppliedEnergistics/Applied-Energistics-2/blob/0387d30d1cbb0c86a3bc5997938fa27e6f831cea/src/main/java/appeng/me/cache/PathGridCache.java#L70) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=findbugs%3AURF_UNREAD_PUBLIC_OR_PROTECTED_FIELD)

**Pull request merged**